### PR TITLE
Don't use RESET_N pin

### PR DESCRIPTION
**Pull request by janakelarsson**
Change comment: Data sheet discourages use of SARA modem reset pin, use may lead to irrecoverable state of modem

### CODE REVIEW

**Comment by janakelarsson** [comment]
@aentinger Here is the change we discussed

File: variants/mkrnb1500/variant.cpp [hunk 1]
```diff
@@ -252,9 +252,10 @@ void initVariant() {
   pinMode(SARA_PWR_ON, OUTPUT);
   digitalWrite(SARA_PWR_ON, LOW);
 
-  // put SARA modem in reset on start to conserve power if it's not used
+  // data sheet discourages use of SARA modem reset pin
+  // use may lead to irrecoverable state of modem
   pinMode(SARA_RESETN, OUTPUT);
-  digitalWrite(SARA_RESETN, HIGH);
+  digitalWrite(SARA_RESETN, LOW);
```
Comment (aentinger) [inline variants/mkrnb1500/variant.cpp:258]: Hi @janakelarsson :wave: This line is already in [ArduinoCore-samd:master:HEAD](https://github.com/arduino/ArduinoCore-samd/blame/0f502c7f193b0a51040c36eaacb36a333a5ad44d/variants/mkrnb1500/variant.cpp#L257) - please rebase your commit.

Comment (janakelarsson) [reply variants/mkrnb1500/variant.cpp:258]: Better now?

Comment (janakelarsson) [reply variants/mkrnb1500/variant.cpp:258]: Or do I need to squash the commits?

Comment (aentinger) [reply variants/mkrnb1500/variant.cpp:258]: Good now, I'll squash them myself.

**Pull request merged**